src/Versions/V2_2_1/Client/Sender.php

```diff
@@ -90,4 +91,13 @@ public function sessions(): Sessions
 
         return $this->sessions;
     }
+
+    public function commands(): Commands
+    {
+        if (!isset($this->commands)) {
```

Comment:
Hello,
You have forgot to add the `commands` property as well. Don't mind, I have added it after merging your PR.